src/Okta.Sdk.IntegrationTests/GroupScenarios.cs

```diff
@@ -119,10 +119,224 @@ public class GroupScenarios : ScenarioGroup
                 () => client.Groups.GetGroupAsync(createdGroup.Id));
         }
 
-        [Fact(Skip = "TODO")]
-        public async Task GroupUserOperations()
+        [Fact]
+        public async Task AddUserToGroup()
         {
-            throw new NotImplementedException();
+            var client = GetClient();
+
+            // Create group
+            var createdGroup = await client.Groups.CreateGroupAsync(new CreateGroupOptions
+            {
+                Name = "New Users Test Group",
+            });
+
+            // Create a user
+            var createdUser = await client.Users.CreateUserAsync(new CreateUserWithPasswordOptions
+            {
+                Profile = new UserProfile
+                {
+                    FirstName = "John",
+                    LastName = "Get-User",
+                    Email = "john-get-user@example.com",
```

Comment:
Rename email and login to `john-add-group@example.com` (these different emails are intended to reduce collisions if the IT doesn't clean up after itself properly for some reason)